src/store/index.ts

```diff
@@ -484,6 +485,14 @@ const {
 
       commit.setNetworkMetadata(networkMetadata);
       commit.setColumnTypes(networkMetadata);
+
+      // Guess the best label variable and set it
+      const allVars: Set<string> = new Set();
+      network.nodes.map((node: Node) => Object.keys(node).forEach((key) => allVars.add(key)));
```

Comment:
Seems like this should be a `forEach()` rather than a `map()`.

Reply:
See 415b69d